ci/ray_ci/tests.windows.env.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rayproject/buildenv:windows
```

Comment:
could you make this filename shorter? why does it need to have 4 words?

maybe create a directory just for windows under `ray_ci` or something?